tests/test_compilers_default.py

```diff
@@ -162,7 +161,22 @@ def test_align_delay_measurement(platform, delay):
 
     target_sequence = PulseSequence()
     if delay > 0:
-        target_sequence[platform.qubits[0].probe.name].append(Delay(duration=delay))
-    target_sequence.extend(platform.qubits[0].native_gates.MZ.create_sequence())
+        target_sequence.append((platform.qubits[0].probe.name, Delay(duration=delay)))
+    target_sequence.concatenate(platform.qubits[0].native_gates.MZ.create_sequence())
     assert sequence == target_sequence
     assert len(sequence.probe_pulses) == 1
+
+
+def test_align_multiqubit(platform):
```

Comment:
Not for this PR, and is also true for existing tests, but this will try to run it using all different platforms in `dummy_qrc`. Currently all except dummy and zurich are disabled, but if we re-enable we may get issues if some platform (eg. rfsoc) doesn't have the `(0, 2)` pair. However, I believe we are already planning to improve the testing suite post 0.2, so I'd leave this for that time.

Reply:
Yes, in general, we should restrict some tests to run only on dummy.
We can make a `dummy` fixture, or parameterize the `platform` one as follows:
```suggestion
@pytest.mark.platforms("dummy", ...)
def test_align_multiqubit(platform):
```
or even return a platform factory, instead of the platform itself (given the way it is used, I expect the `dummy` option to be the most practical).

This is not strictly required for the first release, but I should move this as part of an issue, to avoid forgetting.

Reply:
Now in #977